packages/rich-text/src/remove-line-separator.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+
+import { LINE_SEPARATOR } from './special-characters';
+import { isCollapsed } from './is-collapsed';
+import { remove } from './remove';
+
+/**
+ * Removes a line break character into a Rich Text value at the current
+ * indices. If the Any content between `startIndex` and `endIndex` will be
+ * removed. Indices are retrieved from the selection if none are provided.
```

Comment:
Might there be a typo in this sentence? `If the Any content between startIndex and endIndex will be removed`